src/main/resources/templates/home.html

```diff
@@ -3,84 +3,82 @@
 <head th:include="main.html::head"></head>
 <body>
-<header th:replace="main.html::global-header"></header>
-<main id="main" role="main">
+  <header th:replace="main.html::global-header"></header>
+  <main id="main" role="main">
     <div th:replace="main.html::phase"></div>
     <div class="grid-row">
-        <div class="column-two-thirds">
-            <div class="homepage-intro" th:utext="${registerText}">The register of UK government registers.</div>
-            <dl class="primary-metadata">
-                <dt>Managed by:</dt>
-                <dd><th:block th:include="main.html :: organisation(${registry}, ${branding})"></th:block></dd>
-                <dt>Total records:</dt>
-                <dd><a href="/records"><th:block th:text="${totalRecords}">Total</th:block><span class="visuallyhidden"> records</span></a></dd>
-                <dt>Total entries:</dt>
-                <dd><a href="/entries"><th:block th:text="${totalEntries}">Total</th:block><span class="visuallyhidden"> entries</span></a></dd>
-                <dt>Last updated:</dt>
-                <dd th:text="${lastUpdatedTime}">Last updated</dd>
-            </dl>
-        </div>
+      <div class="column-two-thirds">
+        <div class="homepage-intro" th:utext="${registerText}">The register of UK government registers.</div>
+        <dl class="primary-metadata">
+          <dt>Managed by:</dt>
+          <dd><th:block th:include="main.html :: organisation(${registry}, ${branding})"></th:block></dd>
+          <dt>Total records:</dt>
+          <dd><a href="/records"><th:block th:text="${totalRecords}">Total</th:block><span class="visuallyhidden"> records</span></a></dd>
+          <dt>Total entries:</dt>
+          <dd><a href="/entries"><th:block th:text="${totalEntries}">Total</th:block><span class="visuallyhidden"> entries</span></a></dd>
+          <dt>Last updated:</dt>
+          <dd th:text="${lastUpdatedTime}">Last updated</dd>
+        </dl>
+      </div>
     </div>
     <div class="technical-information">
-        <div class="grid-row">
-            <div class="column-two-thirds">
-            <h2 class="heading-large">Technical information</h2>
-                <p>Registers are made up of entries containing fields of data. To build services on top of these registers you can access the data in a variety of formats.</p>
-                <h3>Fields in this register:</h3>
-                <div class="field-list"> <ul><li th:each="field : ${register.fields}" th:text="${field}"></li></ul> </div>
+      <div class="grid-row">
+        <div class="column-two-thirds">
+          <h2 class="heading-large">Technical information</h2>
+          <p>Registers are made up of entries containing fields of data. To build services on top of these registers you can access the data in a variety of formats.</p>
+          <h3>Fields in this register:</h3>
+          <div class="field-list"> <ul><li th:each="field : ${register.fields}" th:text="${field}"></li></ul> </div>
```

Comment:
I think we can spread this out over multiple lines, it's a looping construct so would be good to increase visibility.